### PR TITLE
Add lesson 13

### PR DESCRIPTION
**Pull request by carlotrimarchi**

### CODE REVIEW

**Comment by carlotrimarchi** [comment]
I'll add a few exercises and homeworks later

File: lesson12.md [hunk 1]
```diff
@@ -119,258 +119,3 @@ let a = {
...
-```js
-let contentElement = document.getElementById('myContent');
-contentElement.style.backgroundColor = "red";
-```
-
----
-
-### Let's practice
-
-Create a HTML page with a button element:
```
Comment (sevtapim) [inline lesson12.md:361]: Create an HTML page with a button element:

File: lesson12.md [hunk 2]
```diff
@@ -119,258 +119,3 @@ let a = {
...
-
-```html
-<body>
-  <button id="myButton">Click me!</button>
-</body>
-```
-
-Now try to set a HTML attribute from JavaScript:
```
Comment (sevtapim) [inline lesson12.md:369]: Now try to set an HTML attribute from JavaScript:

File: lesson13.md [hunk 3]
```diff
@@ -0,0 +1,413 @@
...
+let nameElement = document.getElementById("myName");
+let newName = inputElement.value;
+nameElement.textContent = newName;
+```
+
+---
+
+### Let's practice
+
+Create a HTML page with a button element:
```
Comment (sevtapim) [inline lesson13.md]:
```suggestion
Create an HTML page with a button element:
```

File: lesson13.md [hunk 4]
```diff
@@ -0,0 +1,413 @@
...
+
+```html
+<body>
+  <button id="myButton">Click me!</button>
+</body>
+```
+
+Now try to set a HTML attribute from JavaScript:
```
Comment (sevtapim) [inline lesson13.md]:
```suggestion
Now try to set an HTML attribute from JavaScript:
```

**Review by sevtapim — APPROVED** [review]

**Pull request merged**